etc/kayobe/kolla.yml

```diff
@@ -267,7 +267,7 @@ kolla_enable_murano: false
 #kolla_enable_neutron_agent_ha:
 #kolla_enable_neutron_bgp_dragent:
 #kolla_enable_neutron_provider_networks:
-#kolla_enable_nova_serialconsole_proxy:
+kolla_enable_nova_serialconsole_proxy: true
```

Comment:
Not sure exactly what's involved with this.

Reply:
Hopefully just a redeploy of nova.